community/tools/ra-data-hasura/src/index.js

```diff
@@ -32,6 +30,32 @@ export default (serverEndpoint, headers) => {
       throw new Error(JSON.stringify({'error': 'Invalid table/schema resource'}));
     }
 
+    return {schema: schema, tableName: tableName};
+
+  };
+
+  const getPrimaryKey = (resource) => {
+
+    let primaryKey = 'id';
```

Comment:
Suggestion: It'd be nice if you can define `id` as a constant somewhere at the top of this file.

Something like the follows above L10 would be excellent:

```javascript
const DEFAULT_PRIMARY_KEY = 'id';
```

This way, you can DRY up your code without having to re-write 'id' every time.